<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RaportinmuokkausDialogi</class>
 <widget class="QDialog" name="RaportinmuokkausDialogi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Muokkaa raportin riviä</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="KieliListWidget" name="nimike"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Laskenta</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QRadioButton" name="summaRadio">
            <property name="text">
             <string>Summa</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QRadioButton" name="otsikkoRadio">
            <property name="text">
             <string>Otsikko</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QRadioButton" name="valisummaRadio">
            <property name="text">
             <string>Välisumma</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QCheckBox" name="tyhjaCheck">
            <property name="text">
             <string>Näytä tyhjä</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QCheckBox" name="laskevalisummaanCheck">
            <property name="text">
             <string>Laske välisummaan</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Tiliväli(t)</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="tilitEdit"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Muoto</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Sisennys</string>
            </property>
            <property name="buddy">
             <cstring>sisennysSpin</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="sisennysSpin">
            <property name="maximum">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="valiCheck">
            <property name="text">
             <string>Väli ennen</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="lihavoiCheck">
            <property name="text">
             <string>Lihavointi</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="erittelyCheck">
         <property name="text">
          <string>Erittely</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KieliListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">kieli/kielilistwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RaportinmuokkausDialogi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RaportinmuokkausDialogi</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>summaRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>erittelyCheck</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valisummaRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>tilitEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>summaRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>laskevalisummaanCheck</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>409</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valisummaRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>tyhjaCheck</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
